content/en/docs/setup/_index.md

```diff
@@ -12,84 +12,101 @@ content_template: templates/concept
+| [APPUiO](https://appuio.ch/)  | &#x2714;     | &#x2714;  |   &#x2714; | |  | |
+| [Microsoft Azure](https://azure.microsoft.com)    |  [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/services/kubernetes-service/)     |  |     |        |    | 
+| [CenturyLink Cloud](https://www.ctl.io/)     |       | &#x2714; |   |  | | 
+| [Cloud Foundry Container Runtime (CFCR)](https://docs-cfcr.cfapps.io/)       |     |  |   | &#x2714; |&#x2714; | 
+| [CloudStack](https://cloudstack.apache.org/)           |      |  |   | | &#x2714;| 
+| [Canonical](https://www.ubuntu.com/kubernetes/docs/quickstart)      |              | &#x2714;       |             | &#x2714;     |&#x2714;  | &#x2714;
+| [Containership](https://containership.io/containership-platform)            | &#x2714;       |&#x2714;  |     |     |                   | 
+| [Digital Rebar](https://provision.readthedocs.io/en/tip/README.html)            |        |  |      |     |  | &#x2714;
+| [DigitalOcean](https://www.digitalocean.com/products/kubernetes/)        | &#x2714;      |  |      |     |            | 
+| [Docker Enterprise](https://www.docker.com/products/docker-enterprise) |       |&#x2714;  | &#x2714;     |  |     | &#x2714;
+| [Fedora (Multi Node)](https://kubernetes.io/docs/getting-started-guides/fedora/flannel_multi_node_cluster/)     |      |  |      | |    &#x2714;       | &#x2714;
+| [Fedora (Single Node)](https://kubernetes.io/docs/getting-started-guides/fedora/fedora_manual_config/)     |      |  |      | |           | &#x2714;
+| [Gardner](https://gardener.cloud/)       |              |&#x2714;  |         | &#x2714;   |          | 
+| [Giant Swarm](https://giantswarm.io/)     | &#x2714;       | &#x2714; |   &#x2714;    |     | 
+| [Google](https://cloud.google.com/)           |  [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/)       | [Google Compute Engine (GCE)](https://cloud.google.com/compute/)|[GKE On-Prem](https://cloud.google.com/gke-on-prem/)       |  |  |     |  |      |       |  | 
+| [IBM](https://www.ibm.com/in-en/cloud)        | [IBM Cloud Kubernetes Service](https://cloud.ibm.com/kubernetes/catalog/cluster)| | | [IBM Cloud Private](https://www.ibm.com/in-en/cloud/private)|
+| [Kontena Pharos](https://www.kontena.io/pharos/)          |    |&#x2714;|    &#x2714;    |        |         |
+| [Kubermatic](https://www.loodse.com/)         |     &#x2714;  | &#x2714; | &#x2714;     |     |  |
+| [KubeSail](https://kubesail.com/)    | &#x2714;       |  |      |     |        | 
+| [Kubespray](https://kubespray.io/#/)       |       |    |       |&#x2714;  |  |  |
+| [Kublr](https://kublr.com/)        |&#x2714; | &#x2714;  |&#x2714;       |&#x2714;  | |  |
+| Madcore.ai (404 and no official website)         |              |        |      |    |                      |
+| [Mirantis Cloud Platform](https://www.mirantis.com/software/kubernetes/)    |       |  | &#x2714; |  |    |
```

Comment:
Do you want to include Madcore.ai?

Reply:
The link to Madcore.ai leads to a 404 and I dont see an official website for Madcore.ai. The link to docs leads to a private repo (https://github.com/madcore-ai) was last updated in April 2018. So I removed it.

As you might have already noticed that lot of content in the big table on Picking the right... page is redundant and the columns such as config mgmt, networking , and OS don't add much value in the table because eventually the links to website would anyways provide that information.

Some of the pointers that worked on while doing the re-organization were:
-To eliminate the links that were pointing to vendor specific documentation listed under Turnkey Cloud Solutions sections, Custom Cloud Solutions, and On-Premises VMs. The links now point to vendor websites.
-To ensure that we are vendor neutral by removing the solution descriptions for all vendors.
- Created a decision matrix diagram that depicts the difference in each of the solutions in terms of requirements and available resources for relevant abstractions in K8s. Enables the users to pick the solution that suits their requirements.

Yes, it might be a good idea to have a key to the column definitions. However, I also do not want to have redundant information. The idea to the SVG image was to explain the different solutions based on the abstractions visually rather than a verbose definition. I am open to alternatives....

Reply:
@Rajakavitha1, Looked over the page(s) again. Generally, things look good. 👍 
I think the `Pick..` page still offers details that the `Setup` page does not convey, and the language is easy to read. I like that the main table details the configurations and the OSes supported. I would have to trawl through the company pages to get this information.
The Setup page, to me, reads differently, defining `production` and `learning` environments and seems much more company/solution focused than suited to learners.

Perhaps for a follow-up PR:
- make the table headers a larger font (h3 or h4)
- in the BIG table, capitalize `On-prem`?
- What is `Custom (cloud)`, a Custom datacenter, not on-prem?
- Remove `Madcore` links in the `Pick` page
- I admit that I don't fully understand the new image of production solutions. Who is the user? The expertise depends upon the type of user/company? I like the explanations of the abstractions or categories, and links to the glossary.

Reply:
Thanks @kbhawkey !!!! Yes, may be a follow up PR for the table headers.
I just fixed the capitalization in the table and Yes, Custom (cloud) is a Custom (datacenter)

This PR was raised to address: Addresses #9710
 "Currently, the page is a wall of links. It does not give much explanation for each of the main solution types, nor does it give any rationale for why you would choose one over the other. For example, why would someone choose a hosted solution over a turnkey cloud solution? Consequently, it does not actually help users decide what is best for their needs, which is reflected in the analytics as a high number of page views but low average time on page."

Tried to provide a decision matrix (rationale for why you would choose one over the other) in the production solutions image. For example,
- if you want to opt for kubernetes solution but lack the expertise and resources. Managed solution is what you would opt because most of the abstractions are managed by the provider and you could focus on your application. It is really a matter of choice.
 I was hugely inspired by https://twitter.com/wswmuc/status/681217518778871810 

Do we still want to retain the Picking the Right Solution page when we covered the list of all vendors in this page?

@zacharysarah Please let me know if we should include the  configurations and the OSes columns in the table that are vendor specific.

Reply:
LGTM 👍